ext/php5/ddshared.c

```diff
@@ -0,0 +1,13 @@
+#include "ddshared.h"
+
+#include "container_id/container_id.h"
+#include "ddtrace.h"
+#include "logging.h"
```

Comment:
I'm tired, so maybe it's just me, but I don't see any use of logging in this file. I think it can be removed here and in the versions too.

Reply:
Nice catch. This was left over from the old log message.